src/coreclr/src/vm/interopconverter.cpp

```diff
@@ -111,6 +179,28 @@ IUnknown *GetComIPFromObjectRef(OBJECTREF *poref, ComIpType ReqIpType, ComIpType
     if (*poref == NULL)
         RETURN NULL;
 
+    if (TryGetComIPFromObjectRefUsingComWrappers(*poref, &pUnk))
+    {
+        hr = S_OK;
+        void *pvObj;
+        if (ReqIpType & ComIpType_Dispatch)
+        {
+            hr = pUnk->QueryInterface(IID_IDispatch, &pvObj);
```

Comment:
I assume `TryGetComIPFromObjectRefUsingComWrappers()` is going to `AddRef()`. What about the implicit `AddRef()` in the QI? Does `pvObj` need to be changed to a `ComHolder<T>`?